.travis.yml

```diff
@@ -2,4 +2,4 @@ language: python
 python:
   - "3.6"
 install: pip -q install tox
-script: tox
+script: tox --recreate
```

Comment:
Travis always spins up a new container for each CI run. When does `--recreate` here have an effect?